src/MoBi.Core/OSPSuite.Core/Domain/Builder/CoreExpressionProfile.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OSPSuite.Core.Domain.Services;
+using static OSPSuite.Core.CoreConstants.ContainerName;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class CoreExpressionProfile : BuildingBlock<ExpressionParameter>
```

Comment:
Oh and this is not that here. This now makes the expressionPArameters  as list. We want to be able to access them by path too using a cache
StartValueBuildingBlock<T> is the one (look at PSVBB and MSVBB)

Reply:
you are saying to use StartValueBuildingBlock<T> as the base here? On promotion we can rename this base class PathAndValueBuildingBlock or something like that?

Reply:
Done